main.cpp

```diff
@@ -31,29 +31,308 @@ Purpose:  This project continues developing Project3.
  8) After you finish, click the [run] button.  Clear up any errors or warnings as best you can.
  
  */
-
+#include <iostream>
+using namespace std;
```

Comment:
Do not use `using namespace std`
that's the easiest way to get weird class name collisions when you use other frameworks.
If you're going to use it, limit the scope of its usage to inside functions.
- [ ] revise